src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -142,43 +142,43 @@
 ;;; Public
 
 (schema/defn ^:always-validate
-  initialize-config :- jruby-schemas/JRubyPuppetConfig
+  initialize-config :- jruby-schemas/JRubyConfig
   [config :- {schema/Keyword schema/Any}]
-  (-> (get-in config [:jruby-puppet])
+  (-> (get-in config [:jruby])
```

Comment:
It seems like maybe what we might want to do is check first for the existence of a `:jruby` key in the config, and if it doesn't exist, then fall back to `:jruby-puppet`?  That way we have a path forward without breaking backward compat?